app/lib/project/MetapolatorProject.js

```diff
@@ -655,50 +645,56 @@ define([
                                this.fontinfoFileName, this.fontinfoFile );
     };
 
-    function exportInstance(io, project, masterName, dirName, precision) {
-        // returns a models/Controller
-        var model = project.open(masterName)
-          , master = model.query('master#' + masterName)
-          , glyphSet
-          , exportController
-          ;
-
-        // create a bare ufoV2 directory
-        io.mkDir(false, dirName);
-
-        // create dirName/metainfo.plist
-        io.writeFile(false, dirName+'/metainfo.plist'
-                                , plistLib.createPlistString(metainfoV2));
-
-        // fontforge requires a fontinfo.plist that defines unitsPerEm
-        io.writeFile(false, dirName+'/fontinfo.plist'
-                                , plistLib.createPlistString(minimalFontinfo));
+    _p.exportUFOInstance_chunk = function(masterName, dirName, precision) {
```

Comment:
Forgive me the pun, but this interface it pretty chunky. When `UFOExportController` is fixed, you would have a generator here, which we can also pass directly to the ui code. I think we don't have to keep an instance of the `UFOExportController` around either, the state is in the generator anyways. So, I will look further and think about how we can cash out the generated data ...